### PR TITLE
update known issues

### PR DESCRIPTION
**Pull request by yboussard**
Update for known issues

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -107,6 +107,9 @@ See the [offical Upgrade Guide](http://docs.mattermost.com/administration/upgrad
 * Do not modify the Listen Address in Service Settings.
 * Rarely 'app' container fails to start because of "connection refused" to
   database. Workaround: Restart the container.
+* On Docker version 1.11.2 and 1.11.1 you can have some problems with mattermostdocker_db image (see [issue/26](https://github.com/mattermost/mattermost-docker/issues/26), [issue/6](https://github.com/mattermost/mattermost-docker/issues/6).
+
+  Please use docker version [1.9.1](https://yum.dockerproject.org/repo/main/centos/7/Packages/)  if you can. There is apparently no problem with this version and postgres image
```
Comment (it33) [inline README.md]: Thanks @yboussard, appreciate your help trying out mattermost-docker, sorry to hear about the compatibility issue. Here is some proposed wording to simplify this known issue for the benefit of users: 

```
[The 3.0.2 release on RHEL 7.2 has compatibility issues with Docker version 1.11 and mattermostdocker_db image](https://github.com/mattermost/mattermost-docker/issues/26). RHEL 7.2 with Docker version [1.9.1](https://yum.dockerproject.org/repo/main/centos/7/Packages/) has been verified to work properly with 3.0.2 release. 
```

Please let me know your thoughts?

Comment (yboussard) [inline README.md]: ok. no trouble for me.

Comment (npcode) [inline README.md]: @yboussard Could you please update this pull request as @it33 said?

Comment (cybershambles) [inline README.md]: Docker 1.12 on Centos 7.2 works without issue.

**Comment by esethna** [comment]
Closing this PR for inactivity. Seems this are working as expected on Docker 1.12. Thanks!

**Pull request closed without merging**